addons/website_livechat/controllers/main.py

```diff
@@ -66,3 +67,12 @@ def get_session(self, channel_id, anonymous_name, previous_operator_id=None, cha
         if visitor_sudo:
             anonymous_name = visitor_sudo.with_context(lang=visitor_sudo.lang_id.code).display_name
         return super(WebsiteLivechat, self).get_session(channel_id, anonymous_name, previous_operator_id=previous_operator_id, chatbot_script_id=chatbot_script_id, **kwargs)
+
+    @http.route('/im_livechat/load_templates', type='json', auth='none', cors="*")
+    def load_templates(self, **kwargs):
+        res = super(WebsiteLivechat, self).load_templates(**kwargs)
+        templates = [
+            'website_livechat/static/src/legacy/widgets/public_livechat_floating_text_view/public_livechat_floating_text_view.xml',
+        ]
+        res2 = [tools.file_open(tmpl, 'rb').read() for tmpl in templates]
+        return res + res2
```

Comment:
It would be better to split original method and only override the part providing the templates, and not everything.

And even better it should come from assets bundle, but this might be out of scope of current PR.